crs_application/include/crs_application/common/config.h

```diff
@@ -50,7 +50,7 @@ struct MotionPlanningConfig
 {
   // home pose
   bool pre_move_home;
-  std::vector<std::string> joint_names;
+  std::vector<std::string> home_joint_names;
```

Comment:
Do these only get used for home moves?

Reply:
Yes, this variable is specifically for the home position in joint space. There is also a joint position for media changes. We can have those both be the same position, but I figured it was easy enough to just add a new variable for the media change position. I also had to make sure the media change position was a joint position rather than a Cartesian one because the UR scripts require the robot be in the exact joint position rather than the exact Cartesian position.